os/os.proto

```diff
@@ -303,6 +349,9 @@ message ActivateError {
     // There is no OS package with the version requested for activation. This is
     // also used for an empty version string.
     NON_EXISTENT_VERSION = 1;
+    // Systems which activate on all Supervisors will return NOT_SUPPORTED_ON_BACKUP
+    // when asked to activate on standby Supervisor only.
+    NOT_SUPPORTED_ON_BACKUP = 2;
```

Comment:
Dual Supervisor Target which requires installing the entire system with one Install RPC MUST return NOT_SUPPORTED_ON_BACKUP error when requested to Activate on standby Supervisor.